src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -5,101 +5,93 @@ excerpt: "xk6-browser brings browser automation and end-to-end testing to k6 whi
+| debug             | boolean  | `false` | All CDP messages and internal fine grained logs will be logged if set to `true`.                                                                                                                                                                                      |
+| devtools          | boolean  | `false` | Open up developer tools in the browser by default.                                                                                                                                                                                                                    |
+| env               | string[] | `null`  | Environment variables to set before launching browser process.                                                                                                                                                                                                        |
+| executablePath    | string   | `null`  | Override search for browser executable in favor of specified absolute path.                                                                                                                                                                                           |
+| headless          | boolean  | `true`  | Show browser GUI or not.                                                                                                                                                                                                                                              |
+| ignoreDefaultArgs | string[] | `null`  | Ignore any of the default arguments included when launching browser process.                                                                                                                                                                                          |
+| proxy             | string   | `null`  | Specify to set browser's proxy config.                                                                                                                                                                                                                                |
+| slowMo            | string   | `null`  | Slow down input actions and navigation by the specified time e.g. `'500ms'`.                                                                                                                                                                                          |
+| timeout           | string   | `'30s'` | Default timeout to use for various actions and navigation.                                                                                                                                                                                                            |
+
+### Example
```

Comment:
Apart from a basic example, I think we also need to add a section on how to run the test and some basic trouble shooting guide (e.g. make sure that the binary is placed in the correct environment variable)

Reply:
Wow, how did we forget that 😅 .

I've been working on moving over the `BrowserType` class docs into their own page, and I've already added a "Your First Test" section to the landing page in that [PR](https://github.com/grafana/k6-docs/pull/788) too. For now, i've added a very quick how to run the example test in that same [PR](https://github.com/grafana/k6-docs/pull/788/commits/368f3eed9539e4166a2078366defac7d10d7a10f). It definitely needs more work and we need a troubleshooting page too. It might be better to create an issue to tackle this properly though.

Reply:
Thanks Ankur! I've created this issue to track that we need to add a troubleshooting section  👍🏼 https://github.com/grafana/k6-docs/issues/792